other/verify_image_k8s.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: verify-image-k8s
+  annotations:
+    policies.kyverno.io/title: Verify Image
+    policies.kyverno.io/category: Sample
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/minversion: 1.4.2
+    policies.kyverno.io/description: >-
+      Using the Cosign project, OCI images may be signed to ensure supply chain
+      security is maintained. Those signatures can be verified before pulling into
+      a cluster. This policy checks the signature of an image repo called
+      ghcr.io/kyverno/test-verify-image to ensure it has been signed by verifying
+      its signature against the provided public key. This policy serves as an illustration for
+      how to configure a similar rule and will require replacing with your image(s) and keys.
+spec:
+  validationFailureAction: enforce
+  background: false
+  rules:
+    - name: verify-image
+      match:
+        resources:
+          kinds:
+            - Pod
+      verifyImages:
+      - image: "ghcr.io/kyverno/test-verify-image:*"
+        key: default/cosign-keys
```

Comment:
Where is this? How is it set? Did this exist in 1.4.2? Does it require prerequisites to use this policy?

Reply:
yes, users might generate key pairs first by specifying k8s:// schema while using the `cosign` generate-key-pair command.

@JimBugwadia can you help me a bit, is [PR](https://github.com/kyverno/kyverno/pull/2607/) released? I couldn't see this PR in the changelogs of the releases in Kyverno?

Reply:
This will be a 1.6.0 feature